core/deliverservice/deliveryclient.go

```diff
@@ -128,14 +128,25 @@ func (d *deliverServiceImpl) StartDeliverForChannel(chainID string, ledgerInfo b
 		return errors.Errorf("failed to create block deliverer for channel `%s`, missing OrdererConfig", chainID)
 	}
 
+	createCftDeliverer := false
 	switch ct := oc.ConsensusType(); ct {
 	case "etcdraft":
-		d.blockDeliverer, err = d.createBlockDelivererCFT(chainID, ledgerInfo)
+		createCftDeliverer = true
 	case "BFT":
-		d.blockDeliverer, err = d.createBlockDelivererBFT(chainID, ledgerInfo)
+		switch d.conf.DeliverServiceConfig.Policy {
+		case "cluster", "":
```

Comment:
at this stage the policy should not be "", as we det the default "cluster" much earlier.

Reply:
it is needed for the unit tests, they do not go through the code that "patches" this case, so we need the code here in the delivery client itself.

Reply:
Yes, but it unit test you should check that if the policy is not set you get an error... IMO

Reply:
fixed :)